src/storage/read_pool.rs

```diff
@@ -17,6 +19,7 @@ impl<R: FlowStatsReporter> PoolTicker for FuturePoolTicker<R> {
     }
 }
 
+/// Create respective thread pools to handle read commands of different priority levels
```

Comment:
```suggestion
/// Build respective thread pools to handle read commands of different priority levels
```

Reply:
Trying to follow the verb of the function name makes the comment more readable.

Reply:
Changed.